### DISCUSSION TITLE
Configuring Authorization on Keycloak

### DISCUSSION THREAD

**Discussion by securemationCode** [keycloak/keycloak#13199] [Q&A]
I am using Keycloak version 19.0.3 and have done the authorization configuration using role policies. I am using keycloak.json file on my resource server to enforce the policies. But it is not working as intended. It worked with one policy and one resource only and when I added another resource every user is getting 403 forbidden message. When I evaluate the policies on Keycloak admin console I am getting the expected result but while accessing the resource server it is returning 403. Can someone help or guide me to resolve the issue I am facing ?

**Comment by AndriusKarpavicius** [comment]
You can not repeat same URL over multiple resources. 

If you paste your resource, permission and policy definitions, it would be easier to help.
